infra/k8s/cluster.yaml

```diff
@@ -40,8 +40,19 @@ spec:
     anonymousAuth: false
     maxPods: 200
     allowedUnsafeSysctls:
-      - net.core.somaxconn
       - net.netfilter.nf_conntrack_max
+      - net.core.somaxconn
+      - net.core.netdev_max_backlog
+      - net.core.rmem_max
+      - net.core.wmem_max
+      - net.ipv4.ip_local_port_range
+      - net.ipv4.tcp_max_syn_backlog
+      - net.ipv4.tcp_tw_recycle
+      - net.ipv4.tcp_tw_reuse
+      - net.ipv4.tcp_rmem
+      - net.ipv4.tcp_wmem
+      - net.ipv4.tcp_max_orphans
```

Comment:
@raulk I plugged those 2 as allowed just in case - `tcp_max_orphans` and `tcp_abort_on_overflow`